MethodRealization.cpp

```diff
@@ -0,0 +1,97 @@
+#include "Goro.h"
```

Comment:
Name of file doesn't represent which methods are implemented, please rename this file.